force-app/main/default/classes/SOQL.cls

```diff
@@ -213,14 +311,19 @@ public with sharing class SOQL {
      */
     public static Set<Id> getCaseIdsByAccount(Id accountId) {
         // Create a set to hold the Case Ids
+        Set<Id> caseIds = new Set<Id>();
+
 
         // Query the Cases
+        List<Case> caseList = [SELECT Id
+                            FROM Case
+                            WHERE AccountId = :accountId];
```

Comment:
Here's a trick for getting a set of IDs in one line of code (avoiding the loop and list to set conversion below):
```suggestion
        Set<Case> caseIds = new Map<Id, Case>([SELECT Id
                            FROM Case
                            WHERE AccountId = :accountId]).keySet();
```
Can you see what's happening on the right. First it converts to a map, and then the keys are pulled out and assigned to the set. This works because the keys are a set of IDs. Let me know if you have questions.